wiki/en/Running-a-Server.md

```diff
@@ -83,48 +83,150 @@ If you want to run a number of Servers, possibly also behind a firewall or on a
-| Mono      | Low    | 534 Kbit/s | 306 Kbit/s | 190 Kbit/s | 132 Kbit/s |
+### Installation
 
-## Network bandwidth
+1. Download the [latest headless .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }})
+1. Update apt to make sure you have a current list of standard packages: `sudo apt update`
+1. Install the Jamulus package: `sudo apt install ./{{ site.download_file_names.deb-headless }}`
+1. Enable the headless Server process via systemd: `sudo systemctl enable jamulus-headless`
+1. Add your desired [command line options](Running-a-Server#command-line-options) to the `ExecStart` line in the systemd service file in `/lib/systemd/system/jamulus-headless.service` (By default you will be running an Unregistered Server).
+1. Reload the systemd files `sudo systemctl daemon-reload` and restart the headless Server: `sudo systemctl restart jamulus-headless`
+1. Check all is well with `systemctl status jamulus-headless` (hit `q` to get back to the command prompt).
 
-There is one upstream (musician sending to the Server) and one downstream (server sending back the mix to the musician)
-<figure>
-	<img src="{% include img/en-screenshots/bandwidth-diagram.inc %}" loading="lazy" alt="A diagram of Jamulus network bandwidths from different audio qualities ranging from low to high">
-<figcaption>Calculate bandwidth use </figcaption>
-</figure>
+You may also be interested in downloading [this set of useful tools](https://github.com/jamulussoftware/jamulus/tree/master/tools) from the Jamulus repository (clone the Git repo and also call `git submodule update --init`).
+
+### Configuration
+
+You can control Jamulus with the `systemctl` command. For example, to stop the Server cleanly:
+
+`sudo systemctl stop jamulus-headless`
+
+#### Running in Registered mode
+
+The following minimum setup is required to [run a Registered Server](Running-a-Server#server-types):
+
+~~~
+jamulus --nogui --server \
+        --directoryserver genreServer:port \
+        --serverinfo "yourServerName;yourCity;[country ID]"
+~~~
+
+**Note**: Semicolon and newline characters are not allowed in `--serverinfo`
+
+Replace `genreServer:port` in the example above with one of the following options:
+
+
+| Genre |   Server address           |
+|-----------|------------------|
+|**Any Genre 1** |`anygenre1.jamulus.io:22124`|
+|**Any Genre 2** |`anygenre2.jamulus.io:22224`|
+|**Any Genre 3** |`anygenre3.jamulus.io:22624`|
+|**Genre Rock** |`rock.jamulus.io:22424`|
+|**Genre Jazz** |`jazz.jamulus.io:22324`|
+|**Genre Classical/Folk** |`classical.jamulus.io:22524`|
+|**Genre Choral/Barbershop** |`choral.jamulus.io:22724`|
+
+
+#### Running as a Directory
+
+If you wish to run a [Custom Directory](Running-a-Server#3-custom-directory) please see [this guide](Custom-Directories).
+
+### Maintainance
+
+#### Upgrading
+
+To upgrade your Server to a newer version, simply download a new .deb and re-install as step 3.
```

Comment:
"step 3" here refers back to the list in the _previous_ heading 3 section - at least link to that heading.